Continuous Delivery and Operation Excellence/Continuous Delivery.md

```diff
@@ -0,0 +1,54 @@
+# Content gathered for Continuous Delivery
+
+## Delivery formats
+* Webinar 
+* Self-service content 

```

Comment:
What sort of self-service content? E.g. MS Learn Module, Documents, Videos?

Reply:
It is just a list of planned formats. See details below.